x-pack/legacy/plugins/siem/public/pages/case/components/all_cases/index.test.tsx

```diff
@@ -129,6 +134,39 @@ describe('AllCases', () => {
         .text()
     ).toEqual('Showing 10 cases');
   });
+  it('should render empty fields', () => {
```

Comment:
We should test the component at initial render.

Reply:
this test contains subcomponents which proves the render of the initial component